default.ascx

```diff
@@ -1,67 +1,110 @@
+					<div class="container">
+                        <div class="row">
+                            <div class="logo col col-6 col-lg-3 flex-vert-align">
+                                <!-- #include file="Includes/logo.ascx" -->
+                            </div>
+                            <div class="col d-lg-none text-end d-flex justify-content-end align-items-center">
+                                <button id="navbar-toggler" class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><i class="fa fa-bars"></i></button>
+                            </div>
+							<!-- Top navigation-->
+							<div class="col-12 col-lg">
+								<nav class="navbar navbar-expand-lg">
+									<div class="collapse navbar-collapse" id="navbarSupportedContent">
+										<dnn:MENU MenuStyle="assets/menus/topmenu" runat="server" ExcludeNodes="Admin,Host"></dnn:MENU>
+									</div>
+								</nav>
+							</div>
+						</div>
+					</div>
+                </header>
+            </div>
+                <!-- #include file="Includes/panes.ascx" -->
+                <footer class="skin-footer bg-blue-super-dark skew-top">
+                    <div class="container py-5">
+                        <div class="row gx-5">
+                            <div class="col">
+                                <div class="pane footer_pane_1 col" id="footer_pane_1" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_2 col" id="footer_pane_2" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_3 col" id="footer_pane_3" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_4 col" id="footer_pane_4" runat="server">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="pane footer_pane_full_width pb-0" id="footer_pane_full_width" runat="server">
+                    </div>
+                    <div class="footer-bottom">
+                        <div class="container pt-2">
+                            <div class="row">
+                            <!-- #include file="Includes/footer-bottom.ascx" -->
+                            </div>
+                        </div>
+                    </div>
+                </footer>
+            </div>
+ 
+        <!-- Bootstrap core JS-->
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
I'm confused as to why this is being loaded here from a CDN and why it is a different version of Bootstrap from what is in the package.json.  It appears it is already being compiled from SCSS into `skin.css` in the root, so it seems to be getting loaded at least twice that I have found so far in the theme.

Reply:
I had build issues before and switched to the alternative package for speed. I'll change to the official one

Reply:
This still appears to be an issue - just moved to a new line.

Reply:
@david-poindexter 
Hmm, were you looking at edit.ascx maybe and not default.ascx? 
Seems corrected in default.ascx in the last commit.. (and locally)

![image](https://github.com/DNNCommunity/DNNCommunityTheme/assets/8999382/09b2e556-3710-4391-b3b0-dce541e72bd1)

Reply:
Yes, looks like this was resolved in 18da88be2782609866e098e2a715f5f03a16db4c - thanks!